_posts/2016/2016-09-23-rnd-day-2016.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: "ePages R&D Day 2016"
+date: "2016-09-23 06:08:00"
+image: blog-header/roadmap.jpg
+categories: events
+authors: ["Birgit"]
+---
+
+Last week, we hosted another R&D Day at ePages, that provided great insights into the future of our software.
+This year's focus was clearly on new technologies.
+Developers from all company locations gathered together in Hamburg to present their latest development results as well as important tasks that are in the pipeline for the upcoming months.
+
+We kicked off with a breakfast to prepare for the day ahead and strengthen us for our presentations on stage.
+It was also a nice occasion for first chats and networking talks with colleagues from other teams and locations.
+
+## On stage
+
+During the course of the day we had 7 talks in total on various topics such as "Kubernetes quick dive-in", "Faster to the cloud using modern Java frameworks", "Multistore refurbished" or "Consistent language & localized UI".
+Here's just a quick overview of the keynote presentations.
+
+### Architecture
+
+Our Team Black started off with a general overview of the new ePages architecture.
+They presented the current status of the features they implemented during the last months as well as the roadmap for what they're approaching next.
+Their work emphasizes on a microservice architecture, where they build and enhance a complete new API and software backend as well as a new UI for our merchants.
+With the help of tools and technologies such as [Docker](https://www.docker.com/), [Kubernetes](http://kubernetes.io/), [Kibana](https://www.elastic.co/products/kibana), [Zipkin](http://zipkin.io/) and various [Spring](https://spring.io/) projects they currently implement all required functionalities for a complete user journey.
+
+### Storefront, administration, editor
+
+Next on stage was our Team Unity with a presentation of the current status of our new responsive storefront and editor as well as the reworked shop administration.
```

Comment:
Could be misleadingly read as if the new editor is also responsive, which it is not.